### PR TITLE
Add ID tooling, remove redundant scripts

### PR DESCRIPTION
**Pull request by tplooker**
- Add basic tooling for managing this spec as an ID draft based on https://github.com/martinthomson/internet-draft-template.
- Removes some redundant scripts for old tooling.
- Removes old HTML version.
- Minor tweaks to draft naming based on #30

### CODE REVIEW

File: .github/update.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: "Update generated files"
+# This rule is not run automatically.
```
Comment (OR13) [inline .github/update.yml:2]: remove comments?

File: .github/setup.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        with:
+          make: update-venue
+
+      - name: "Update GitHub Pages"
+        if: ${{ steps.first.outputs.first != 'true' }}
+        uses: martinthomson/i-d-template@v1
+        with:
+          make: gh-pages
+
+      - name: "Push Changes"
```
Comment (OR13) [inline .github/setup.yml:88]: Looks like you are no longer using markdown2rfc.... I had to bump the version recently... https://github.com/OR13/markdown2rfc

I wanted to render latex equations as svgs...

Comment (OR13) [reply .github/setup.yml:88]: I love the idea of github handling this all for you tho, i would still probably want to test it locally.

File: .github/update.yml (hunk 1 above)
Comment (tplooker) [reply .github/update.yml:2]: It just describes how the tool is run copied directly from the template (https://github.com/martinthomson/internet-draft-template/blob/main/.github/workflows/update.yml#L2)

File: .github/setup.yml (hunk 2 above)
Comment (tplooker) [reply .github/setup.yml:88]: Yeah I have seen some other repos that include the ability to do this locally to preview changes in final output form, was going to tackle in a follow up PR

File: .github/setup.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
+name: "Perform Initial Repository Setup"
```
Comment (tplooker) [inline .github/setup.yml:1]: Have updated this workflow file essentially it is designed to be run once then deletes it self, just sets up tooling around the draft publication

File: .github/setup.yml (hunk 2 above)
Comment (tplooker) [reply .github/setup.yml:39]: Note as above this workflow is just designed to be run once also

**Comment by tplooker** [comment]
Merging PR so we can get spec hosting working correctly, no substantive changes to the spec text.

**Pull request merged**